modules/ROOT/pages/introduction/intro-config-use-fd.adoc

```diff
@@ -0,0 +1,142 @@
+* Limits the number of reconnection attempts to three at a frequency of every 5000 milliseconds (ms)
++
+.Use the fields in the Reconnection Strategy section to specify reconnection options
+image::introduction/intro-config-overview-reconnection-strategy-fd.png[Reconnection strategy example using Flow Designer]
+. To use TLS, scroll down and specify the required information in the *TLS Configuration* section.
+. Click *Save*.
+
+For more information about configuring `HTTP Listener`, see the xref:http/http-documentation.adoc[HTTP Connector Reference].
+
+[[scheduler]]
+=== Use Scheduler as an Input Source
+
+Use `Scheduler` to trigger a flow when a time-based condition is met. You can configure `Scheduler` to trigger a flow at a regular interval or give it a more flexible cron expression.
+
+To use `Scheduler` as an input source:
+
+. Follow the procedure described in <<configure-input-source,Configure an Input Source for the Flow>>.
+. In the *Search* field, enter `Scheduler`.
+. Select *Scheduler*.
+. In the *Scheduling Strategy* field, select whether you want to use a fixed frequency or cron expression to trigger the flow:
++
+For example, the following configuration directs `Scheduler` to:
+
+* Wait 10 seconds before initiating the flow for the first time after the app starts
+* Initiate the flow every 30 seconds after the initial 10 second delay
++
+.Select `Fixed Frequency` in the *Scheduling Strategy* field to initiate a flow at regular intervals.
+image::introduction/intro-config-fd-scheduler-fixed.png[Configuration for a fixed scheduling strategy]
++
+
+The following configuration uses a cron expression to log `my message` every second:
++
+.Select `Cron` in the *Scheduling Strategy* field to use a cron expression to specify when to initiate a flow.
+image::introduction/intro-config-fd-scheduler-cron.png[Configuration using a cron expression]
+
+For more information about configuring `Scheduler`, see xref:mule-runtime::scheduler-concept.adoc[Scheduler Endpoint]
+
+
+[[connector-specific]]
+=== Use a Connector-Specific Input Source
+
+If a connector has associated input sources, you can use a connector input source to initiate the flow. For example, you can use `Amazon SQS Receive Messages` as an input source for Anypoint connector for Amazon SQS (Amazon SQS Connector). This input source initiates a flow each time the configured Amazon SQS queue receives a new message.
+
+To start the configuration for connector-specific input sources, follow these steps:
+
+. Follow the procedure described in <<configure-input-source,Configure an Input Source for the Flow>>.
+. In the *Trigger* field, search for and select the connector.
+. If the connector has multiple input sources, select the input source to use. (If the connector has one input source, that input source is selected automatically.)
+
+Each connector-specific input source has a unique configuration, but most configurations require the following actions:
```

Comment:
What does "has a unique configuration" mean? Does it mean that each trigger operation is already configured by default and that the default configuration is unique for that trigger operation? If so, I'm still not sure what "unique" means.

Reply:
Disagree on this one.